maven/ReleaseAndStage/Jenkinsfile

```diff
@@ -37,8 +37,8 @@ mavenNode {
     }
 
     echo 'NOTE: running pipelines for the first time will take longer as build and base docker images are pulled onto the node'
-    container(name: 'maven') {
-      stage('Build Image') {
+    container(name: 'maven', shell:'/bin/bash') {
+      stage('Build Release') {
```

Comment:
you need to change it

Reply:
Why does this need to change?  It worked in OSIO with the fabric8 image and ran correctly.